scss/core/elements/input/_input.compact.scss

```diff
@@ -94,7 +93,27 @@
 			& > label {
 				@include flex-order(-1);
 			}
+
+			& > #{$prefix}.input#{$selector} {
+				margin-right: 0px
```

Comment:
Besides, why is this specific to compact inputs ? Shouldn't it be shared between both styles ?